pkg/store/store.go

```diff
@@ -0,0 +1,21 @@
+// Package store provides a simple persistent atomic store for usage and
+// quota.
+package store
+
+import "errors"
+
+type Value struct {
+	SizeBytes int64
+}
+
+var ErrNotFound = errors.New("not found")
+
+type Store interface {
+	// Get returns the value associated with key.
+	Get(key string) (Value, error)
+	// Set associates value with key.
+	Set(key string, value Value) error
+	// AddSizeBytes adds to the SizeBytes field of the Value associated
+	// with key.  It creates a new blank Value if needed.
+	AddSizeBytes(key string, numBytes int64) error
```

Comment:
Since the event from S3 gives the new size, shouldn't this be "SetSizeBytes"?

(In case of lakeFS the behavior will be the same because we do not overwrite files, but for other Terminus users it may be desired to replace the size rather than add it. Unless object-level versioning is turned on, in which case the current behavior makes sense.)